tasks/ilo.yml

```
@@ -1,34 +1,33 @@
 ---
 # vim:ft=ansible:
 
- - assert: { that: "ansible_system_vendor == 'HP'" }
+- assert: { that: "ansible_system_vendor == 'HP'" }
 
-   # We look for:
-   # 01:00.0 "System peripheral" "Hewlett-Packard Company" "Integrated Lights-Out Standard Slave Instrumentation & System Support" -r05 "Hewlett-Packard Company" "iLO4"
- - name: Checking for ILO device
-   command: lspci -s 01:00.0 -m
-   register: ilo_info
-   changed_when: False
-   check_mode: no
+- name: Checking for ILO device
+  shell: hponcfg -w conf.txt|grep -i 'Firmware Revision'| awk '{print $8$9}'
+  register: ilo_info
+  changed_when: False
+  check_mode: no
 
-   # We need to get the "iLO4" from the above into a varible
-   # using the python split fuction on the '"' character and the 2nd from last
-   # element of the resulting array. "|lower" is a Jinja2 filter
- - name: Getting ilo version
-   set_fact: ilo_version={{ ilo_info.stdout.split('\"')[-2] |lower}}
+- debug:
+    msg: "{{ ilo_info }}"
 
- - assert: { that: ilo_version is match("ilo4") }
```

Why remove this? I like to fail early.